protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3305,3 +3314,66 @@ message PingRequest {
 // Ping response from the data node
 message PingResponse {
 }
+
+// Basic description of an order.
+message OrderInfo { 
+ // Side for the order, e.g. buy or sell.
+  vega.Side side = 1;
+  // Price for the order. The price is an integer, for example `123456` is a correctly
+  // formatted price of `1.23456` assuming market configured to 5 decimal places.
+  // This field is an unsigned integer passed as a string and needs to be scaled using the market's decimal places.
+  string price = 2;
+  // Size remaining, when this reaches 0 then the order is fully filled and status becomes filled.
```

Comment:
Was this comment copied from elsewhere? I'm not sure if this bit makes sense in this context:
```
when this reaches 0 then the order is fully filled and status becomes filled.
```

Also the field is a `uint64` but the doc says:
"// This field is an unsigned integer **passed as a string**"